.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```

Comment:
I wouldn't commit IDE/Text editor config like this to github. It isn't really needed. You can use a gitignore file to stop git tracking files like this.